app-text/texlive-core/texlive-core-2023-r9.ebuild

```diff
@@ -396,7 +396,7 @@ src_test() {
 		-e 's;dvispc.test;;' \
 		texk/dviout-util/Makefile || die
 
-	emake check
+	emake check -j1
```

Comment:
This should have a comment pointing to the but that caused the restriction to `-j1`.

That said, upstream told us that we may be able to grab the wcfname.test scripts from current and run the tests with texlive 2023. See https://bugs.gentoo.org/935825#c3

We should try this first to before restricting ourselves to run all tests sequentially.